### PR TITLE
Added unencoded private tx example 

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added unencoded private tx example

### CODE REVIEW

File: docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md [hunk 1]
```diff
@@ -1,26 +1,76 @@
...
+passed as an input parameter to [`eea_sendRawTransaction`](../../Reference/API-Methods.md#eea_sendrawtransaction).
+
+To create a [Besu-extended private transaction](../../Concepts/Privacy/Privacy-Groups.md#besu-extended-privacy), 
+specify a `privacyGroupId` when creating the signed transaction 
+passed as an input parameter to [`eea_sendRawTransaction`](../../Reference/API-Methods.md#eea_sendrawtransaction).
+
+## Unsigned and unencoded private transactions 
+
+The [`eea_sendRawTransaction`](../../Reference/API-Methods.md#eea_sendrawtransaction) parameter is a 
+signed RLP-encoded private transaction. Examples of unsigned and unencoded private transactions are
```
Comment (NicolasMassart) [inline docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md]: maybe we could clarify the process of signing/encode a tx (link to the page where it's explained) from the unsigned content bellow.

File: docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md [hunk 2]
```diff
@@ -1,26 +1,76 @@
...
+
+The [`eea_sendRawTransaction`](../../Reference/API-Methods.md#eea_sendrawtransaction) parameter is a 
+signed RLP-encoded private transaction. Examples of unsigned and unencoded private transactions are 
+displayed below. 
+
+!!! example "Unencoded and unsigned EEA-compliant private transaction"
+ 
+     ```
+     {
+       "from": "0xfe3b557e8fb62b89f4916b721be55ceb828dbd73",
```
Comment (iikirilov) [inline docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md]: missing `to` and `nonce` from the examples. `to` is `null` if it is a contract creation transaction

File: docs/Reference/API-Methods.md [hunk 3]
```diff
@@ -3967,10 +3967,11 @@ None
 
 ### eea_sendRawTransaction
 
-Creates a [private transaction](../Concepts/Privacy/Privacy-Overview.md) from a signed transaction, generates the transaction hash 
-and submits it to the transaction pool, and returns the transaction hash of the Privacy Marker Transaction.
+Distributes the [private transaction](../HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md), 
+generates the transaction hash and submits it to the transaction pool, and returns the transaction hash of the Privacy Marker Transaction.
```
Comment (iikirilov) [inline docs/Reference/API-Methods.md]: I think this is a little confusing. Is this the transaction hash of the private transaction? No - it is the transaction hash of the privacy marker transaction that is sent to the transaction pool. And then link to the part of the doc where you define PMT.

File: docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md]: Added a tip on this

File: docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md]: Added.  Can you confirm that null doesn't have quotes? ie, the following is correct

```
{
       "to": null,
       "from": "0xfe3b557e8fb62b89f4916b721be55ceb828dbd73",
       "gas": "0x7600", 
       "gasPrice": "0x0", 
       "data": "0x608060405234801561001057600080fd5b5060dc8061001f6000396000f3006080604052600436106049576000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff1680633fa4f24514604e57806355241077146076575b600080fd5b348015605957600080fd5b50606060a0565b6040518082815260200191505060405180910390f35b348015608157600080fd5b50609e6004803603810190808035906020019092919050505060a6565b005b60005481565b80600081905550505600a165627a7a723058202bdbba2e694dba8fff33d9d0976df580f57bff0a40e25a46c398f8063b4c00360029", 
       "nonce": "0x0",
       "privateFrom": "negmDcN2P4ODpqn/6WkJ02zT/0w0bjhGpkZ8UP6vARk=",
       "privateFor": ["g59BmTeJIn7HIcnq8VQWgyh/pDbvbt2eyP0Ii60aDDw=","6fg8q5rWMBoAT2oIiU3tYJbk4b7oAr7dxaaVY7TeM3U="],
       "restriction": "restricted"
     }
```

File: docs/Reference/API-Methods.md (hunk 3 above)
Comment (MadelineMurray) [reply docs/Reference/API-Methods.md]: Fixed

File: docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md (hunk 2 above)
Comment (iikirilov) [reply docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md]: confirmed

**Pull request merged**